src/Form/FormAnalyzer.js

```diff
@@ -34,8 +34,16 @@ class FormAnalyzer {
          */
         this.signals = []
 
+        // Analyse the input that was passed. This is pretty arbitrary, but historically it's been working nicely.
         this.evaluateElAttributes(input, 1, true)
-        form ? this.evaluateForm() : this.evaluatePage()
+
+        // If we have a meaningful container (a form), check that, otherwise check the whole page
+        if (form !== input) {
```

Comment:
This is a minor fix that regressed at some point. Largely inconsequential, but we were supposed to check if the input is === the form, instead we were checking for form existence. Nothing changed in the test suite, so it wasn't doing any harm. It's an edge case anyway. I've also expanded the conditional for clarity.